RangeShiftR/NAMESPACE

```diff
@@ -19,6 +20,9 @@ export(SMSpathLengths)
 export(Settlement)
 export(Simulation)
 export(StageStructure)
+export(Translocation)
+export(createODD)
```

Comment:
I also included the ODD feature again in this version. The description of the translocation feature still needs some work